core/src/client/client.rs

```diff
@@ -277,6 +277,52 @@ impl AssetClient for Client {
             Ok(None)
         }
     }
+
+    fn is_asset_spent(
+        &self,
+        transaction_hash: H256,
+        index: usize,
+        shard_id: ShardId,
+        block_id: BlockId,
+    ) -> TrieResult<Option<bool>> {
```

Comment:
Would you add a comment about the meaning of output?

Reply:
I added the comment explaining the case of None.